Disable saved searches

awead opened the pull request:
A feature of Blacklight allows users to save searches for future reference; however, we don't use this and a side-effect is that the searches table can grow excessively large. This disables any searches from being saved into the table.

Fixes #857

cam156 commented on lib/tasks/scholarsphere/redis.rake:
```diff
@@ -0,0 +1,20 @@
+            Redis.current.del("events:#{id}")
+          end
+        end
+        Redis.current.rename(key, key.gsub(/GenericFile/, "GenericWork"))
```
@awead does this need to be sufia:

awead replied:
Ah, yes, I believe you're right.

awead committed:
Disable saved searches

A feature of Blacklight allows users to save searches for future
reference; however, we don't use this and a side-effect is that the
searches table can grow excessively large. This disables any searches
from being saved into the table.

awead commented:
@whereismyjetpack this will prevent any further records from being written, but we'll still need to truncate the table after this is deployed. I figure we can just do that in console.
